src/runtime-rs/crates/resource/src/manager_inner.rs

```diff
@@ -237,6 +237,17 @@ impl ResourceManagerInner {
         self.cgroups_resource.delete().await
     }
 
+    pub async fn cleanup_hostpath(&self) -> Result<()> {
+        if let Some(share_fs) = &self.share_fs {
+            share_fs
+                .get_share_fs_mount()
+                .cleanup_hostpath(&self.sid)
```

Comment:
Suggest providing a `cleanup()` for  the `share_fs` and encapsulated `get_share_fs_mount().cleanup_hostpath(&self.sid)` in the `cleanup()`.

Reply:
fixed